src/components/Mana/Mana.tsx

```diff
@@ -46,5 +58,18 @@ export class Mana extends React.Component<ManaProps & HeaderProps> {
         {children}
       </Header>
     )
+
+    if (showTooltip) {
```

Comment:
Just as a small comment, we usually write: `return showTooltip ? <Popup {...props} /> : ManaInner`.
Not mandatory though IMO.